### PR TITLE
Allow schema to be passed in resource

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

File: odo/backends/sql.py [hunk 1]
```diff
@@ -422,21 +427,52 @@ def append_select_statement_to_sql_Table(t, o, **kwargs):
     return t
 
 
-@resource.register('(.*sql.*|oracle|redshift)(\+\w+)?://.+')
+def create_schema(ddl, target, bind, tables=None, state=None, checkfirst=None,
```
Comment (llllllllll) [inline odo/backends/sql.py]: This is not `create_schema` but `has_schema`, no?

Comment (cpcloud) [inline odo/backends/sql.py]: Good call. Will change when back from lunch.

**Comment by cpcloud** [comment]
@llllllllll Merge-able?

**Comment by llllllllll** [comment]
lgtm

**Pull request merged**